log/log_test.go

```diff
@@ -222,6 +222,9 @@ func TestLogTraceableError(t *testing.T) {
 func TestFailSeek(t *testing.T) {
 	err := ArchiveLogFile("archivefile")
 	if err == nil {
+		defer func() {
+			_ = os.Remove("./archivefile")
```

Comment:
In theory, this is not needed as `archivefile` should not get created unless there is a bug in the code.
If we are leaving this, please drop the "./" so the filename is consistent.

Reply:
Hmm ..looks like the file remains after a make check.

Reply:
I don't think your defer should be inside the if
Perhaps before the call to ArchiveLogFile

Reply:
Done